estuary/utils/recents.py

```diff
@@ -46,7 +46,10 @@ def get_recent_nodes():
             node_results = final_result_data.setdefault(label, [])
             # result is always a list of a single node
             node = inflate_node(result[0])
-            node_results.append(node.serialized)
+            serialized_node = node.serialized
+            serialized_node['resource_type'] = node.__label__
+            serialized_node['display_name'] = node.display_name
```

Comment:
In case these fields are "None", are we just gonna show "None"? How was it before? Is there maybe a way to show a more useful information?

Reply:
I think for the label they will never be none because it is just based off of what type of node/artifact it is. As for the display name, I'm not sure. @mprahl thoughts?

Reply:
Yes, each model class that this API endpoint will use implements a property method of `display_name`. If they don't, a `NotImplementedError` exception is raised from the base class.